### PR TITLE
Arithnea Logo on Adopters page

### PR DESCRIPTION
**Pull request by dBagaria**
**Description**

Changes proposed in this pull request:

- Added Arithnea Logo as per the add new user guide

**Related issue(s)**

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
@dBagaria Hi! Can you check CLA? Without this, we cannot merge your PR :(

**Comment by dBagaria** [comment]
Hi, just signed the CLA. Thanks for the update :)

**Comment by dBagaria** [comment]
Closed the PR by mistake. Reopened it.

File: content/adopters/adopters.yaml [hunk 1]
```diff
@@ -18,3 +18,6 @@ adopters:
   - websiteUrl: "https://faircg.com/"
     logo: "./logos/FAIR_LOGO_HEADER.svg"
     content: "Kyma enables us to quickly develop flexible and robust solutions to extend our clients cloud-based and on-premise enterprise applications."
+  - websiteUrl: "https://www.arithnea.de/"
+    logo: "./logos/arithnea.svg"
+    content: "Kyma supports us with tools to create unique customer experiences with SAP CX even more efficient, flexible and at low risk."
```
Comment (kazydek) [inline content/adopters/adopters.yaml]:
```suggestion
    content: "Kyma supports us with tools to create a unique customer experience with SAP CX - one that is even more efficient, flexible, and at low risk."
```

**Comment by magicmatatjahu** [comment]
@dBagaria Can you recheck CLA and push to PR suggestion from @kazydek ? We can't wait for merge it :D

**Comment by dBagaria** [comment]
Hi, to sign the CLA i clicked on the Details link next to the check and was redirected to cla-assistant page there after clicking on the "Sign-in with Github to agree" button I get the message that the CLA has been signed and i am getting redirected back to Github. I however see that the license/cla chaeck is still pending. Am i doing something wrong? 

image

**Comment by dBagaria** [comment]
There seems to be some problem with the first commit that i made and it seems like that is the reason why the CLA is not working. I am closing this pull request and will create another one shortly. Thank you for your support

**Pull request closed without merging**